build/Dockerfile

```diff
@@ -1,17 +1,16 @@
-# Dockerfile to generate CentOS 8 build container for XMLTV
+# Dockerfile to generate Fedora 40 build container for XMLTV
 # Part 1
-# Source: https://github.com/xmltv/xmltv/blob/master/ci/Dockerfile.centos-8
+# Source: https://github.com/xmltv/xmltv/blob/master/ci/Dockerfile.fedora-40
 
+FROM fedora:40
 
-FROM centos:8
+LABEL maintainer="Nick Morrott <knowledgejunkie@gmail.com>"
```

Comment:
Thank you for your contribution! Could you please remove this line?
Nick Morrott is not the maintainer of the this Dockerfile. While he is the maintainer of the base Dockerfile we use, we already refer to the source.